src/index.js

```diff
@@ -1,4 +1,4 @@
-if (process.env.NODE_ENV !== 'production') require('source-map-support').install()
+try { require('source-map-support').install() } catch (e) { }
```

Comment:
I think this should work natively if sources are built with `--source-maps` in babel?